examples/2d/plot_scattering_disk.py

```diff
@@ -1,102 +1,191 @@
+#
+# Morlet wavelets $\psi_{\theta}$ are Hermitian, i.e: $\psi_{\theta}^*(u) = \psi_{\theta}(-u) = \psi_{\theta+\pi}(u)$.
+#
+# As a consequence, the modulus wavelet transform of a real signal $x$ computed with a Morlet wavelet $\psi_{\theta}$
+# is invariant by a rotation of $\pi$ of the wavelet. Indeed, since $(x*\psi_{\theta}(u))^* = x*(\psi_{\theta}^*)(u) =
+# x*\psi_{\theta+\pi}(u)$, we have $\lvert x*\psi_{\theta}(u)\rvert = \lvert x*\psi_{\theta+\pi}(u)\rvert$.
 #
-# Combining this and the fact that a real signal has a Hermitian symmetry
-# implies that it is usually sufficient to use the angles $\{\frac{\pi l}{L}\}_{l\leq L}$ at computation time.
-# For consistency, we will however display $\{\frac{2\pi l}{L}\}_{l\leq 2L}$,
-# which implies that our visualization will be redundant and have a symmetry by rotation of $\pi$.
+# Scattering coefficients of order $n$:
+# $\lvert \lvert \lvert x * \psi_{\theta_1, j_1} \rvert * \psi_{\theta_2, j_2} \rvert \cdots * \psi_{\theta_n, j_n}
+# \rvert * \phi_J$ are thus invariant to a rotation of $\pi$ of any wavelet $\psi_{\theta_i, j_i}$. As a consequence,
+# Kymatio computes scattering coefficients with $L$ wavelets whose orientation is uniformly sampled in
+# an interval of length $\pi$.
 
 L = 6
 J = 3
-scattering = Scattering2D(J=J, shape=src_img.shape, L=L, max_order=1, frontend='numpy')
+scattering = Scattering2D(J=J, shape=src_img.shape, L=L, max_order=2, frontend='numpy')
 
 ####################################################################
 # We now compute the scattering coefficients:
 src_img_tensor = src_img.astype(np.float32) / 255.
 
-scattering_coefficients = scattering(src_img_tensor)
-print("coeffs shape: ", scattering_coefficients.shape)
+scat_coeffs = scattering(src_img_tensor)
+print("coeffs shape: ", scat_coeffs.shape)
 # Invert colors
-scattering_coefficients = -scattering_coefficients
+scat_coeffs= -scat_coeffs
 
 ####################################################################
-# We skip the low pass filter...
-scattering_coefficients = scattering_coefficients[1:, :, :]
-norm = mpl.colors.Normalize(scattering_coefficients.min(), scattering_coefficients.max(), clip=True)
-mapper = cm.ScalarMappable(norm=norm, cmap="gray")
-nb_coeffs, window_rows, window_columns = scattering_coefficients.shape
+# There are 127 scattering coefficients, among which 1 is low-pass, $JL=18$ are of first-order and $L^2(J(J-1)/2)=108$
+# are of second-order. Due to the subsampling by $2^J=8$, the final spatial grid is of size $4\times4$.
+# We now retrieve first-order and second-order coefficients for the display.
+len_order_1 = J*L
+scat_coeffs_order_1 = scat_coeffs[1:1+len_order_1, :, :]
+norm_order_1 = mpl.colors.Normalize(scat_coeffs_order_1.min(), scat_coeffs_order_1.max(), clip=True)
+mapper_order_1 = cm.ScalarMappable(norm=norm_order_1, cmap="gray")
+# mapper of coefficient amplitude to a grayscale color for visualisation
```

Comment:
capital letter, dot

Reply:
Sorry what do you mean ?

Reply:
Ok I did not know the formalism was that demanding! Will change

Reply:
well, the original one wasn't great and i guess i increased my standards